CalibTracker/SiStripChannelGain/src/SiStripGainsPCLWorker.cc

```diff
@@ -267,6 +267,9 @@ void SiStripGainsPCLWorker::processEvent(const TrackerTopology* topo) {
       if(Validation)     {ClusterChargeOverPath/=(*gainused)[i];}
       if(OldGainRemoving){ClusterChargeOverPath*=(*gainused)[i];}
     }
+
+    // keep processing of pixel cluster charge until here
+    if(APV->SubDet<=2) continue;
```

Comment:
hi @dimattia , Question : can't we get rid completely of the pixel part ? there are  have  checks and computation for the pixel above this line like here 
https://github.com/dimattia/cmssw/blob/8bdcb4372e22de2ac5112b37575b3bead83546a5/CalibTracker/SiStripChannelGain/src/SiStripGainsPCLWorker.cc#L262
or here : 
https://github.com/dimattia/cmssw/blob/8bdcb4372e22de2ac5112b37575b3bead83546a5/CalibTracker/SiStripChannelGain/src/SiStripGainsPCLWorker.cc#L238
  which are eventually not used  - Same comments for SiStripGainFromCalibTree.cc  . 
What do you think ? thanks